apps/microsoft-teams/frontend/src/components/config/ContentTypeSelectionModal/ContentTypeSelectionModal.tsx

```diff
@@ -48,26 +81,19 @@ const ContentTypeSelectionModal = (props: Props) => {
       return (
         <>
           <Modal.Content>
+            <DebouncedSearchInput
+              placeholder={searchPlaceholder}
+              onChange={handleSearchQueryUpdate}
+            />
             <FormControl as="fieldset" marginBottom="none">
               <Table className={styles.table}>
                 <Table.Body>
```

Comment:
This is a little confusing how we define a `<Table>` wrapper and then the table rows are delegated to `<SearchableList>`... It's not the end of the world since it's defined in this component, I just wonder if passing `{ children }` would read easier 🤔

Reply:
I don't have a strong opinion here. But I think it works nicely the way you've got it now and you haven't named things that couples it to a specific HTML element choice 👍

Reply:
Sounds good, we can always tweak the implementation later if we want but I'm happy with this solution as is.